src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## Unreleased
+
+## 0.3.0-beta
```

Comment:
@cijothomas said on Gitter it will be 0.4.0-beta because he already pushed 0.3.0-beta as a test? Do we want to start there in these?

Reply:
In Python we've explored https://github.com/CookPete/auto-changelog and it didn't seem to fly.
But I agree with you that we should be lazy and type less code as much as possible.